ingest/expression_files/expression_files.py

```diff
@@ -138,6 +158,66 @@ def check_unique_cells(cell_names: List, study_id, client):
             raise ValueError(error_string)
         return True
 
+    @staticmethod
+    def has_expression_file_info_doc(client, study_id):
+        COLLECTION_NAME = "study_files"
+        QUERY = {"study_id": study_id, "expression_file_info": {"$exists": True}}
+        query_results = list(client[COLLECTION_NAME].find(QUERY))
+        if query_results:
+            return True
+        return False
+
+    @staticmethod
+    def generate_query_filters(
+        query_results, field_values, query_mappings: Dict
+    ) -> List[Dict]:
+        """Creates query filters
+
+        :parameter
+            query_results (List[Dict]) : Results from any query
+            field_values (List) : Keys that are taken from query_results
+            query_mappings : Maps keys in query_results to fields a model
+        Example:
+               query_results = [{_id: 1234}, {food:abc}, {barbara: streisand}]
+               field_values = [id, food]
+               query_mappings = {id: study_file_id, barbara: lastname}
+
+               results = [{study_file_id: 1234}, {lastname: streisand}]
+        """
+        filters = []
+        for results in query_results:
+            for field in field_values:
+                field_key = query_mappings[field]
+                field_value = results[field]
+                filters.append({field_key: field_value})
+        return filters
+
+    @staticmethod
+    def get_study_file_ids_by_type(
```

Comment:
Since we're in the `ExpressionFiles` class, and this method (as written) will only ever return expression matrices: 
```suggestion
    def get_study_expression_file_ids(
```

Or something similar.  Since we're not passing `file_type` as a parameter, it seems somewhat misleading to name the method this.

Reply:
[99313db](https://github.com/broadinstitute/scp-ingest-pipeline/blob/097b087cf1298b0d2143f92d7b748b32232b92c0/ingest/expression_files/expression_files.py#L187)